Userland/Libraries/LibWeb/HTML/HTMLProgressElement.cpp

```diff
@@ -17,4 +19,57 @@ HTMLProgressElement::~HTMLProgressElement()
 {
 }
 
+RefPtr<Layout::Node> HTMLProgressElement::create_layout_node(NonnullRefPtr<CSS::StyleProperties> style)
+{
+    return adopt_ref(*new Layout::Progress(document(), *this, move(style)));
+}
+
+double HTMLProgressElement::value() const
+{
+    auto parsed_value = strtod(attribute(HTML::AttributeNames::value).characters(), nullptr);
+    if (!isfinite(parsed_value) || parsed_value < 0)
+        return 0;
+
+    return min(parsed_value, max());
+}
+
+void HTMLProgressElement::set_value(double value)
+{
+    if (value < 0)
+        return;
+
+    set_attribute(HTML::AttributeNames::value, String::formatted("{}", value));
```

Comment:
Sorry to nitpick, but I did miss commenting on this stringify, my bad :^(

Reply:
I should check all places for this. Fixed :)